### PR TITLE
Add owner to examples and tests

### PR DESCRIPTION
**Pull request by connorwstein**
Making these examples more realistic with an owner and adding tests for them, part of the vrfv2 documentation cleanup.

### CODE REVIEW

**Review by samsondav — APPROVED** [review]

File: core/services/vrf/integration_v2_test.go [hunk 1]
```diff
@@ -381,6 +385,94 @@ func registerProvingKeyHelper(t *testing.T, uni coordinatorV2Universe, vrfkey vr
...
+func TestSimpleConsumerExample(t *testing.T) {
+	owner := newIdentity(t)
+	random := newIdentity(t)
+	genesisData := core.GenesisAlloc{
+		owner.From: {Balance: assets.Ether(10)},
+	}
+	backend := cltest.NewSimulatedBackend(t, genesisData, ethconfig.Defaults.Miner.GasCeil)
+	linkAddress, _, linkContract, err := link_token_interface.DeployLinkToken(
+		owner, backend)
+	require.NoError(t, err)
```
Comment (samsondav) [inline core/services/vrf/integration_v2_test.go:445]: Very minor point, but some whitespacing here to format into conceptual "paragraphs" would make this more readable

**Comment by samsondav** [comment]
@cl-homu r+

**Comment by cl-homu** [comment]
:pushpin: Commit f3cf4090d2d03e94dc94708db31516aae0d81d38 has been approved by `samsondav`

**Comment by cl-homu** [comment]
:hourglass: Testing commit f3cf4090d2d03e94dc94708db31516aae0d81d38 with merge 80d5b9abe2bf4355d9758513fc0306501f433951...

**Comment by cl-homu** [comment]
:broken_heart: Test failed - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4211852272?check_suite_focus=true)

**Comment by cl-homu** [comment]
:sunny: Test successful - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4213096781?check_suite_focus=true), [checks-lint](https://github.com/smartcontractkit/chainlink/runs/4211852115?check_suite_focus=true), [checks-npm](https://github.com/smartcontractkit/chainlink/runs/4213096905?check_suite_focus=true), [checks-operator](https://github.com/smartcontractkit/chainlink/runs/4211852045?check_suite_focus=true), [checks-prettier](https://github.com/smartcontractkit/chainlink/runs/4211852317?check_suite_focus=true), [checks-solidity](https://github.com/smartcontractkit/chainlink/runs/4211852114?check_suite_focus=true)
Approved by: samsondav
Pushing 80d5b9abe2bf4355d9758513fc0306501f433951 to develop...

**Comment by cl-homu** [comment]
:eyes: Test was successful, but fast-forwarding failed: 422 Update is not a fast forward

**Comment by connorwstein** [comment]
@cl-homu retry

**Comment by cl-homu** [comment]
:broken_heart: Test failed - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4214329813?check_suite_focus=true)

**Comment by jmank88** [comment]
@cl-homu retry
:fairy:

**Comment by cl-homu** [comment]
:hourglass: Testing commit f3cf4090d2d03e94dc94708db31516aae0d81d38 with merge 13472c353121191144a766f33ce0ff886b444368...

**Comment by cl-homu** [comment]
:sunny: Test successful - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4215352564?check_suite_focus=true), [checks-lint](https://github.com/smartcontractkit/chainlink/runs/4215352321?check_suite_focus=true), [checks-npm](https://github.com/smartcontractkit/chainlink/runs/4215352690?check_suite_focus=true), [checks-operator](https://github.com/smartcontractkit/chainlink/runs/4215352388?check_suite_focus=true), [checks-prettier](https://github.com/smartcontractkit/chainlink/runs/4215352501?check_suite_focus=true), [checks-solidity](https://github.com/smartcontractkit/chainlink/runs/4215352325?check_suite_focus=true)
Approved by: samsondav
Pushing 13472c353121191144a766f33ce0ff886b444368 to develop...

**Pull request merged**